webpack/components/AnsibleHostDetail/components/AnsibleVariableOverrides/AnsibleVariableOverridesTable.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { translate as __ } from 'foremanReact/common/I18n';
+import {
+  TableComposable,
+  Thead,
+  Tbody,
+  Tr,
+  Th,
+  Td,
+} from '@patternfly/react-table';
+
+import {
+  formatValue,
+  formatSourceAttr,
+} from './AnsibleVariableOverridesTableHelper';
+
+const AnsibleVariableOverridesTable = ({ variables, hostAttrs }) => {
+  const columns = [
+    __('Name'),
+    __('Ansible Role'),
```

Comment:
Should the role be the first column and group the variables from the same role together?

Reply:
No, we do not go for the same design as what we have in host form. The variables from the same role are grouped by default, I imagine we could have sortable header for role in the future.